flask_restplus_patched/namespace.py

```diff
@@ -138,13 +140,13 @@ def dump_wrapper(*args, **kwargs):
                 # pylint: disable=missing-docstring
                 response = func(*args, **kwargs)
 
-                if isinstance(response, flask.Response):
+                if isinstance(response, flask.Response) or model is None:
```

Comment:
Ouch, HTTP/204 tests failed... We need to move this whole `if` after the `elif` (so this `if` will become `elif`, and that `elif` will be `if`).